compiler/src/main/scala/dx/translator/cwl/ProcessTranslator.scala

```diff
@@ -476,6 +476,8 @@ case class ProcessTranslator(cwlBundle: CwlBundle,
       Stage(call.name, getStage(), calleeName, inputs, callee.outputVars)
     }
 
+    // TODO this is where fragment applet is built
```

Comment:
For WDL, it's here
compiler/src/main/scala/dx/translator/wdl/CallableTranslator.scala